### PR TITLE
feat(docker): add kubernetes files

### PR DESCRIPTION
**Pull request by L-U-C-K-Y**
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

I added kubernetes deployment files

- **Why was this change needed?** (You can also link to an open issue here)

Maybe it's helpful to the community

- **Other information**:

Used kustomize and added README.
I added it within the docker folder as it's base is the `docker-compose.yaml`

### CODE REVIEW

**Comment by Cliftonz** [comment]
@L-U-C-K-Y  I would move this deployment of novu to its own kustomize folder, /docker/kubernetes -> /docker/kubernetes/kustomize
This way if there are any other deployments we want to support then we can put them in there own folders for example, /docker/kubernetes/helm.

**Comment by L-U-C-K-Y** [comment]
> @L-U-C-K-Y I would move this deployment of novu to its own kustomize folder, /docker/kubernetes -> /docker/kubernetes/kustomize This way if there are any other deployments we want to support then we can put them in there own folders for example, /docker/kubernetes/helm.

Good point, I have moved the files!

File: docker/kubernetes/kustomize/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+
+# Kubernetes Deployment Files
+
```
Comment (Cliftonz) [inline docker/kubernetes/kustomize/README.md:3]: I would add a prerequisite section here that link to how to install kustomize and set up kustomize.
This is not something everyone knows how to do.

Comment (L-U-C-K-Y) [reply docker/kubernetes/kustomize/README.md:3]: I added a note for kustomize and argo-cd

**Comment by L-U-C-K-Y** [comment]
> Otherwise, I think this is great!

Thanks for your review, happy that others find it helpful, I have added a note for kustomize and argo-cd.

**Review by scopsy — APPROVED** [review]
Looks great @L-U-C-K-Y! Thanks for the contribution here and thanks @Cliftonz for the review 🚀

**Pull request merged**